grails-app/controllers/au/org/ala/merit/ProjectController.groovy

```diff
@@ -918,12 +918,7 @@ class ProjectController {
             if (model.activity.siteId) {
                 model.reportSite = sites?.find { it.siteId == model.activity.siteId }
             }
-
-            Map siteData = projectService.projectSites(projectId)
-            if (!siteData.error) {
-                model.projectArea = siteData.projectArea
```

Comment:
Looks like projectArea is removed here but is used in GSP. And projectArea returned via ajax call is not used.

Reply:
Thanks @temi I've fixed this.